example-1/index.html

```diff
@@ -1,48 +1,67 @@
-    <title>Example 1 - Mollie Components</title>
-    <link rel="stylesheet" href="/base.css" />
-    <link rel="stylesheet" href="./style.css" />
+    <title>Mollie Components - Example 1</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./styles.css" />
   </head>
-  <body>
-    <a class="link-source" target="_blank" rel="noopener noreferrer" href="https://github.com/mollie/components-examples/tree/master/example-1">View source</a>
 
+  <body>
     <div class="wrapper">
       <form method="post" class="form" id="mcForm">
         <div class="form-fields">
-          <div class="form-group form-group--card-holder">
-            <label class="label" for="card-holder">Card holder</label>
-            <div id="card-holder"></div>
-            <div id="card-holder-error" class="field-error" role="alert"></div>
+          <div class="row">
+            <div class="form-group form-group--card-holder">
+              <label class="label" for="card-holder">Name</label>
+              <div id="card-holder"></div>
+              <div
+                id="card-holder-error"
+                class="field-error"
+                role="alert"
+              ></div>
+            </div>
           </div>
 
-          <div class="form-group form-group--card-number">
-            <label class="label" for="card-number">Card number</label>
-            <div id="card-number"></div>
-            <div id="card-number-error" class="field-error" role="alert"></div>
+          <div class="row">
+            <div class="form-group form-group--card-number">
+              <label class="label" for="card-number">Credit card number</label>
+              <div id="card-number"></div>
+              <div
+                id="card-number-error"
+                class="field-error"
+                role="alert"
+              ></div>
+            </div>
           </div>
 
-          <div class="form-group form-group--expiry-date">
-            <label class="label" for="expiry-date">Expiry date</label>
-            <div id="expiry-date"></div>
-            <div id="expiry-date-error" class="field-error" role="alert"></div>
-          </div>
+          <div class="row">
+            <div class="form-group form-group--expiry-date">
+              <label class="label" for="expiry-date">Expiry date</label>
+              <div id="expiry-date"></div>
+              <div
+                id="expiry-date-error"
+                class="field-error"
+                role="alert"
+              ></div>
+            </div>
 
-          <div class="form-group form-group--verification-code">
-            <label class="label" for="verification-code">Verification code</label>
-            <div id="verification-code"></div>
-            <div id="verification-code-error" class="field-error" role="alert"></div>
+            <div class="form-group form-group--verification-code">
+              <label class="label" for="verification-code">CVC</label>
```

Comment:
I get your point, but since these are examples I don't think it matters that much. More important here is the implementation since it's for developers. I think it's safe to assume that they recognize this field as cvv/cvc, what do you think?